Documentation/distributions.md

```diff
@@ -0,0 +1,50 @@
+# How to get rkt on each Linux distribution
+
+## CoreOS
+
+rkt is pre-installed on CoreOS.
+The [CoreOS release page](https://coreos.com/releases/) specifies the version of rkt available in each CoreOS release channel.
```

Comment:
`The [CoreOS releases page](https://coreos.com/releases/) lists the version of rkt available in each CoreOS release channel.`